src/Predis.php

```diff
@@ -51,23 +60,82 @@ public function memoizeCallable(string $key, callable $compute, int $cacheTime =
     {
         if (!$this->refresh) {
             try {
-                $cached = $this->client->get($key);
+                if (rand(1, 100) <= $this->refreshPercent) {
```

Comment:
Should this support checking for 0 if someone never wants to use this functionality?

Reply:
If it's 0 then this will always be false